d2renderers/d2fonts/d2fonts.go

```diff
@@ -26,6 +32,20 @@ func (f FontFamily) Font(size int, style FontStyle) Font {
 	}
 }
 
+func (f Font) GetEncodedSubset(corpus string) string {
```

Comment:
actually why do you even need to get unique chars spread out all over the place when you can just do it once here

Reply:
I don't have access to the diagram here

Reply:
why do u need it? i'm saying u just filter unique chars of corpus

Reply:
also I can't determine here if its an animated svg or not

Reply:
?? i don't think u understand. u still collect chars where ur collecting chars, u just don't filter for uniqueness until here